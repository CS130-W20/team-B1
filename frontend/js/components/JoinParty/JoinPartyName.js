import React from 'react';
// import 'bootstrap/dist/css/bootstrap.css';
// import '../css/MainScreen.css';
import { Button } from 'reactstrap';


const JoinPartyName = () => {
  return (
  	<div className="JoinPartyName">
  	    <div class="title">
	    	<h1>What is your name? </h1>
  		</div>

  		<div class="enterName">
	    	<input type="text"  value= "Name"/>
  		</div>

    	<div className="buttons">
	    	<Button className="button" color="success"> Join party
	    		<small> Lets get rockin' </small>
	    	</Button>
  		</div>
  	</div>

  );
}

export default JoinPartyName;
