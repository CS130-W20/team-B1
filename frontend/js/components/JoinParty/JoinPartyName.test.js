import React from 'react';
import renderer from 'react-test-renderer';

import JoinPartyName from './JoinPartyName';

describe ('Testing Spotify Login Page', () => {
    it('correctly renders', () => {
        const wrapper = renderer.create(<JoinPartyName />).toJSON();
        expect(wrapper).toMatchSnapshot();
    });
});