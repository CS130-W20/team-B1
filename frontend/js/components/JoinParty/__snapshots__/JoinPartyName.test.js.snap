// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Spotify Login Page correctly renders 1`] = `
<div
  className="JoinPartyName"
>
  <div
    class="title"
  >
    <h1>
      What is your name? 
    </h1>
  </div>
  <div
    class="enterName"
  >
    <input
      type="text"
      value="Name"
    />
  </div>
  <div
    className="buttons"
  >
    <button
      aria-label={null}
      className="button btn btn-success"
      onClick={[Function]}
    >
       Join party
      <small>
         Lets get rockin' 
      </small>
    </button>
  </div>
</div>
`;
